splunk-otel-android/src/test/java/com/splunk/rum/PostApi28NetworkDetectorTest.java

```diff
@@ -145,6 +146,7 @@ public void carrierIsSet() {
                 new PostApi28NetworkDetector(
                         connectivityManager, telephonyManager, carrierFinder, context);
         CurrentNetwork currentNetwork = networkDetector.detectCurrentNetwork();
-        assertThat(currentNetwork.getCarrierName()).isEqualTo("flib");
+        assertThat(currentNetwork.getNetworkAttributes())
+                .containsEntry(SemanticAttributes.NET_HOST_CARRIER_NAME, "flib");
```

Comment:
Yeah, so I think this fundamentally shows the problem around changing a data model object to have semantics in a new domain. Meh.